src/Components/ClusterRules/ClusterRules.js

```diff
@@ -244,6 +274,12 @@ const ClusterRules = ({ cluster }) => {
       sortingRows = [...rows].sort((firstItem, secondItem) => {
         const fst = firstItem[0].rule[CLUSTER_RULES_COLUMNS_KEYS[index]];
         const snd = secondItem[0].rule[CLUSTER_RULES_COLUMNS_KEYS[index]];
+        if (index === 3) {
```

Comment:
please don't use hard-coded values here. Could you please map column indices with the analogy to https://github.com/RedHatInsights/ocp-advisor-frontend/blob/acba19655f0b803ee934a2c226b1489170dd2ca5/src/AppConstants.js#L264 ?

then you could use it here in a way:
```suggestion
        if (index === CLUSTER_RULES_IMPACTED_CELL) {
```

Reply:
Great suggestion. Thanks